packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -278,6 +278,8 @@ class AndroidSdk {
   List<AndroidSdkVersion> _sdkVersions;
   AndroidSdkVersion _latestVersion;
 
+  final bool licensesOnly;
```

Comment:
It's not present if you haven't accepted the licenses - in which case gradle wouldn't download the assets for you.

That said, this would probably be more useful as a getter called `platformToolsAvailable` or `missingPlatformTools`.  I'll look at that and make sure it's documented.